### ISSUE TITLE
jakub-onderka/php-console-highlighter outdated

### ISSUE THREAD

**Issue by koromerzhin** [povils/phpmnd#79]
phpmnd v2.0.0 requires jakub-onderka/php-console-highlighter v0.3.2 but this library is outdated. 0.4 version is 12 days ago

**Comment by peter279k** [comment]
According to the `composer.json`, the `jakub-onderka/php-console-highlighter` verdion definition inside `require` block is as follows:

```
......
jakub-onderka/php-console-highlighter: "^0.3.2", ......
```

That means this package require `jakub-onderka/php-console-highlighter` at least `0.3.2` version.

I think we should install `0.4` version to check compatibility.

**Comment by gergelypap** [comment]
+1

**Comment by povils** [comment]
Solved with [#88](https://github.com/povils/phpmnd/pull/88)
